examples/openvino/stable-diffusion/train_text_to_image_qat.py

```diff
@@ -700,7 +650,7 @@ def get_nncf_config(pipeline, dataloader, args):
                 "ignored_scopes": [
                     "{re}.*__add___[0-2]",
                     "{re}.*layer_norm_0",
-                    "{re}.*Attention.*/bmm_0",
+                    # "{re}.*Attention.*/bmm_0",
```

Comment:
get the following error from nncf when left uncommented : 
`No match has been found among the model operations for the following ignored/target scope definitions`